content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Announcing Istio 1.9
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9! Officially supported versions of Kubernetes releases are 1.17-1.20
+for this Istio release.
+
+{{< relnote >}}
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers Shamsher Ansari (RedHat), Steven Landlow (Google) and Jacob Delgado (Aspen Mesh) for helping to get Istio 1.9.0
+published.
+
+Given the work done by the UX Group, notably Mitch Connors (Google) and Christian Posta (Solo.io), and led by the
+direction of the [Istio TOC](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md), Istio 1.9's
+core focus was to improve Day 2 operations. These include stabilizing installation, upgrades, core functionality and
+features, based on the criteria on the [Feature Status](https://istio.io/latest/about/feature-stages/) page. Thank you
+to everyone that participated in the survey in helping to make Istio 1.9 be our most successful release to date.
+
+More information will be published in the future detailing the Istio community's plans for 2021 in an upcoming blog.
+
+Here are some highlights for this release:
+
+## Feature promotion
+
+### Virtual machine installation promoted to Beta
```

Comment:
cc @howardjohn @linsun Please take a closer look at this section. Improvements using the "insert a suggestion" are welcome.

Reply:
can we have some links to docs here?

```
* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
```

Reply:
Added links to the updated docs.